### PR TITLE
Add test-framework dependency to graphics testframework package

### PR DESCRIPTION
**Pull request by marctem**
### Purpose of this PR
In latest trunk, NUnit and Core Test Framework were moved out of trunk into packages. Add a dependency in our Graphics Test Framework package so tests compile and run again.

---
### Testing status
**Katana Tests**: Green! https://katana.bf.unity3d.com/projects/com.unity.render-pipelines/builders?ScriptableRenderLoop_branch=test-framework-deps&unity_branch=trunk&automation-tools_branch=master&sort=1-asc

**Manual Tests**: What did you do?
- [X] Opened test project + Run graphic tests locally

---
### Overall Product Risks
**Technical Risk**: None - fixes broken tests for all SRP packages

**Halo Effect**: None

### CODE REVIEW

File: com.unity.testframework.graphics/package.json [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
-	"displayName":"Graphics Tests Framework",
-	"version": "6.7.0",
-	"unity": "2019.2",
-	"description": "Provides test framework helpers for writing tests for graphics code, such as image comparison assertions and automatic management of reference images.",
-	"keywords": ["qa", "test", "testing", "tests", "graphics"],
-	"dependencies": {
-	}
+    "name": "com.unity.testframework.graphics",
+    "displayName":"Graphics Tests Framework",
+    "version": "6.6.0",
```
Comment (sophiaaar) [inline com.unity.testframework.graphics/package.json]: why was the version set to 6.6.0?

**Comment by sophiaaar** [comment]
comment added here https://github.com/Unity-Technologies/ScriptableRenderPipeline/pull/3481/files#r279785289

File: com.unity.testframework.graphics/package.json (hunk 1 above)
Comment (stramit) [reply com.unity.testframework.graphics/package.json]: not 6.6.6 :p

Comment (stramit) [reply com.unity.testframework.graphics/package.json]: we are already that high 
https://unity.bintray.com/unity-staging/com.unity.testframework.graphics/-/

Comment (marctem) [reply com.unity.testframework.graphics/package.json]: My bad, I didn't mean to drop it down! Must have been my local trunk version. Good catch, I'll fix.

**Comment by sebastienlagarde** [comment]
Bump. Are we doing something for this PR ?

**Comment by marctem** [comment]
@sebastien-unity please merge this. Tests pass except for known LW issues (and the fact that any tests run at all means this PR has no impact). This will simply allow projects under TestProjects to have cleaner manifest files.

**Comment by marctem** [comment]
Not necessary, other fixes made this obsolete while waiting to merge.

**Pull request closed without merging**